docker/Dockerfile.customopbuilder.clean

```
@@ -56,4 +56,12 @@ RUN rm -f /usr/bin/python && \
     pip --version
 
 COPY --from=cuda /usr/local/cuda /usr/local/cuda
-ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
\ No newline at end of file
+ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64:/usr/local/cuda/lib64/stubs/:${LD_LIBRARY_PATH}
+
+WORKDIR /opt/dali
+COPY qa/setup_packages.py qa/setup_packages.py
+RUN export USE_CUDA_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1\2/') &&\
+    export last_config_index=$(python qa/setup_packages.py -n -u tensorflow-gpu --cuda ${USE_CUDA_VERSION}) &&\
```

nitpick (several occurrences) 
```suggestion
    export last_config_index=$(python qa/setup_packages.py -n -u tensorflow-gpu --cuda ${USE_CUDA_VERSION}) && \
```

---

Done